### ISSUE TITLE
Press and Hold Virtual Key

### ISSUE THREAD

**Issue by ItsEngi** [Neos-Metaverse/NeosPublic#2733]
### Is your feature request related to a problem? Please describe.

The Push-To-Talk button is excellent, but there is currently no way for a user to hold down the key in VR. While this will have a permanent solution in the future, it could be easy to add a way for users to create their own solutions in the meantime. 

Holding down a VirtualKey button repeatedly presses the key, behaving as if it were inputting into a text field which makes it unable to function as a push-to-talk button.

### Describe the solution you'd like

A simple boolean value on the VirtualKey component that toggles the behavior of holding it down. By default the behavior would be set to repeat as it behaves now to preserve its use in virtual keyboards. Toggling the boolean would make pressing and holding the button component "hold the key down," only sending an unpressed event when the user stops holding the button down.

In other words, the button could choose between behaving as "pulse this key repeatedly" and "trigger and hold this key until unpressed"

### Describe alternatives you've considered

Much like CommonUI's ButtonToggle, there could also be a VirtualKeyToggle component that, when triggered by an OnPressed button event, latches the key down in the same manner. This would be even more non-invasive to existing keyboards as the VirtualKey component would remain untouched.

### Additional context

This would have some extra knock-on effects as well, such as using the WASD keys in VR to move.

**Comment by shadowpanther** [comment]
If your use-case is toggling the voice mode, there are other ways to do that. For example, using Cloud Variables to pass data from worldspace to userspace and switching voice off of that. No need to emulate key press.
